sdks/python/apache_beam/dataframe/frame_base.py

```diff
@@ -303,20 +309,26 @@ def wrapper(self, *args, **kwargs):
 
 
 def wont_implement_method(msg):
+  """Generate a stub method for `op` that simply raises a WontImplementError.
```

Comment:
Can you please update the comment for the `msg` argument instead of `op`.

Reply:
thanks

```suggestion
  """Generate a stub method that simply raises WontImplementError(msg).
```